css/style.css

```diff
@@ -0,0 +1,544 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  min-width: 1170px;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1030px;
+}
+
+/* Первый блок */
+
+.heading {
+  min-height: 96px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.heading__nav-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.heading__logo {
+  margin-right: 45px;
+}
+
+.navigation__link {
+  text-decoration: none;
+  font-family: Gotham Pro Bold, sans-serif;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #000;
+  margin-right: 35px;
+}
+
+.heading__hamburger-wrapper {
+  display: none;
+}
+
+.search-form__input {
+  height: 20px;
+  width: 185px;
+  border-bottom: 1px solid #000;
+  background-color: #fff;
+  display: flex;
+  margin-right: 35px;
+}
+
+.search-form__text {
+  font-family: Gotham Pro Regular, sans-serif;
+  border: none;
+  outline: none;
+  width: 316px;
+  vertical-align: middle;
+  color: #6c6c6c;
+  font-size: 14px;
+}
+
+.search-form__submit {
+  padding-right: 14px;
+  width: 14px;
+  height: 14px;
+  border: none;
+  background: url(../icons/search.png) no-repeat 50% 50%;
+}
+
+.heading__extras {
+  display: flex;
+  align-items: center;
+}
+
+.heading__signin {
+  text-decoration: none;
+  font-family: Gotham Pro Bold, sans-serif;
+  color: #000;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  margin-right: 35px;
+}
+
+.heading__cart-wrapper {
+  display: flex;
+  align-items: center;
+  margin-right: 35px;
+}
+
+.heading__cart-link {
+  text-decoration: none;
+  font-family: Gotham Pro Bold, sans-serif;
+  color: #000;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  margin-right: 4px;
+}
+
+.heading__cart-img {
+  height: 15px;
+}
+
+.heading__flag {
+  width: 20px;
+}
+
+/* Второй блок */
+
+.main__wrapper {
+  height: 280px;
+  background: url(../img/banner-bg.png);
+  margin-bottom: 54px;
+}
+
+.main__text-wrapper{
+  padding-top: 91px;
+  padding-left: 60px;
+  max-width: 195px;
+}
+
+.main__header {
+  font-family: Gotham Pro Bold, sans-serif;
+  font-size: 30px;
+  line-height: 29px;
+  text-transform: uppercase;
+  color: #FFF;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#FFF" to be "#fff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5575166)